eZ/Publish/Core/MVC/Symfony/Templating/RenderContentStrategy.php

```diff
@@ -39,20 +40,28 @@ public function render(ValueObject $valueObject, RenderOptions $options): string
         $currentRequest = $this->requestStack->getCurrentRequest();
         $surrogateCapability = $currentRequest->headers->get('Surrogate-Capability');
 
-        $request = new Request();
+        $request = Request::createFromGlobals();
```

Comment:
If we're actually going to need all the request information, why aren't we using a clone of current request from request stack instead? Possibly also adjusting it so it is identified as a subrequest, instead of master request?

Reply:
All we actually need is base path (whatever would it be) and there is no way to manually add it to the request. Clone would work here as well, but not sure is it worth doing it.